Controllers/EmployeeManagement/CouriersController.cs

```diff
@@ -63,15 +57,23 @@ public ActionResult<Courier> Get(int id)
         /// <param name="value">new created courier</param>
         /// <returns>one courier by id</returns>
         /// <response code="201">Returns the newly created item</response>
-        /// <response code="404">If the item is null</response>
+        /// <response code="400">If the item is null</response>
         // POST api/<CouriersController>
         [HttpPost]
         public ActionResult<Courier> Post([FromBody] Courier value)
         {
-            if (value != null)
-                return CreatedAtAction("Get", new { id = value.id }, value);
+            _context.Couriers.AddAsync(value);
```

Comment:
Adding to database should happen after validation and asynchronous addition should be awaited